display-p3/image-view/src/main/cpp/simple_png.cpp

```diff
@@ -130,12 +130,17 @@ PNGHeader::PNGHeader(std::string& name, uint8_t *buf, uint64_t len) :
       default:
         LOGV("====Unprocessed CHUNK %c%c%c%c",
              type.bytes[3], type.bytes[2], type.bytes[1], type.bytes[0]);
-        // on purpose: pass through
+        [[fallthrough]];
```

Comment:
this requires C++ 17,  could you change 11 to 17 in [CMakeLists.txt](https://github.com/android/ndk-samples/blob/master/teapots/image-decoder/src/main/cpp/CMakeLists.txt#L40)